pkg/controller/util.go

```diff
@@ -326,3 +329,297 @@ func addToMap(m1, m2 map[string]string) map[string]string {
 	}
 	return m1
 }
+
+// returns the CloneRequest string which contains the pvc name (and namespace) from which we want to clone the image.
+func getCloneRequestPVC(pvc *v1.PersistentVolumeClaim) (string, error) {
+	cr, found := pvc.Annotations[AnnCloneRequest]
+	if !found || cr == "" {
+		verb := "empty"
+		if !found {
+			verb = "missing"
+		}
+		return cr, errors.Errorf("annotation %q in pvc \"%s/%s\" is %s\n", AnnCloneRequest, pvc.Namespace, pvc.Name, verb)
+	}
```

Comment:
I am not sure it's worth all of this code to differentiate the log message between empty or missing.  You could just say

```
errors.Errorf("annotation %q in pvc \"%s/%s\" is empty or missing", AnnCloneRequest, pvc.Namespace, pvc.Name)
```

Reply:
done

Reply:
@aglitke this code comes from the cdi-controller. If we simplify it here was should do the same in cdi-controller. The reason for doing this was thinking there is value in distinguishing between _missing_ vs present but _empty_.